lbrynet/lbrynet_console/LBRYSettings.py

```diff
@@ -1,15 +1,25 @@
 import binascii
+import functools
 import json
-import unqlite
 import logging
 import os
+
 from twisted.internet import threads, defer
+import unqlite
```

Comment:
I looked into that, but unqlite 0.5.3 requires cython and I wasn't sure it was worth the extra complexity in the build process.

But, since it makes the windows build easier, I've added: https://app.asana.com/0/136290697597644/168474034092992